src/assets/_project/_blocks/components/forms/recaptcha.js

```diff
@@ -117,6 +117,8 @@ import keys from '../../data/qg-google-keys';
                   $successMsgContainer.removeClass('d-none');
                   $('#qg-page-feedback-form, .qg-feedback-toggle').addClass('d-none');
                   $successMsgContainer.append($.parseHTML(JSON.parse(response).message));
+                }).fail(function () {
+                  $successMsgContainer.append('<p>Request failed, please try again</p>');
```

Comment:
Check with others what this text should say, unsure if request failed is good enough, for us it is but for the common citizen of qld it might be confusing to them as they won't know what it means.

Reply:
@jackie-QLD 
if the request for submission (new footer feedback ) is failed what should be the message?

Reply:
Thanks @asifamingov  - As discussed, I'm happy with "There was a problem submitting your feedback. Please try again."